### PR TITLE
Create Indicator_Resource, Indicator_Question, Resource models

### PR DESCRIPTION
**Pull request by ppuri96**
#7 #8 #11 models created and tested 

Fixed paradigm model ranking scope and added paradigm relationships.

### CODE REVIEW

File: app/models/paradigm.rb [hunk 1]
```diff
@@ -12,8 +11,7 @@ class Paradigm < ActiveRecord::Base
```
Comment (tchambers3) [inline app/models/paradigm.rb:10]: Validate numericality and only_integer for ranking

File: db/migrate/20170321191838_create_resources.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateResources < ActiveRecord::Migration
+  def change
+    create_table :resources do |t|
+      t.integer :paradigm_id
+      t.string :title
+      t.text :link
+      t.boolean :active
```
Comment (tchambers3) [inline db/migrate/20170321191838_create_resources.rb:7]: default active to true

File: app/models/paradigm.rb (hunk 1 above)
Comment (ppuri96) [reply app/models/paradigm.rb:10]: This is already done, I think git collapsed those lines.

File: db/migrate/20170326175006_fix_indicator_resources.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class FixIndicatorResources < ActiveRecord::Migration
+  def change
+    rename_column :indicator_resources, :indictaor_id, :indicator_id
+    change_column :resources, :active, :boolean, :default => true
```
Comment (tchambers3) [inline db/migrate/20170326175006_fix_indicator_resources.rb]: Remove this line unless the other migration file wasn't working.

**Pull request merged**